### ISSUE TITLE
Bug/crowdsec: Debian package fails to start after hub upgrade

### ISSUE THREAD

**Issue by ThinkChaos** [crowdsecurity/crowdsec#888]
**Describe the bug**
See #716 last comments. Running `cscli hub update && cscli hub upgrade` causes the GeoIP db files to be missing.

**Technical Information (please complete the following information):**
 - OS: Debian
 - Version 1.0.9-2+b4-debian

**Additional context**
Relevant log entry:
```
crowdsec - goroutine crowdsec/runParse crashed : interface conversion: interface {} is nil, not parser.GeoIpEnricherCtx
```

Backtrace:
```
version: 1.0.9-2+b4-debian
Codename: alphaga
BuildDate: 2021-08-04_20:01:32
GoVersion: 1.15.9
goroutine 115 [running]:
runtime/debug.Stack(0xc000011110, 0xc00084c780, 0x5e)
	runtime/debug/stack.go:24 +0x9f
github.com/crowdsecurity/crowdsec/pkg/types.CatchPanic(0x12942e6, 0x11)
	github.com/crowdsecurity/crowdsec/pkg/types/utils.go:100 +0x238
panic(0x11879c0, 0xc0009d0d80)
	runtime/panic.go:969 +0x1b9
github.com/crowdsecurity/crowdsec/pkg/parser.GeoIpCity(0xc00069b790, 0xe, 0xc000818640, 0x0, 0x0, 0x1, 0x1, 0x0)
	github.com/crowdsecurity/crowdsec/pkg/parser/enrich_geoip.go:82 +0x73b
github.com/crowdsecurity/crowdsec/pkg/parser.(*Node).ProcessStatics(0xc000a33268, 0xc000544480, 0x9, 0x9, 0xc000818640, 0x1, 0x1)
	github.com/crowdsecurity/crowdsec/pkg/parser/runtime.go:146 +0x4d5
github.com/crowdsecurity/crowdsec/pkg/parser.(*Node).process(0xc000a33268, 0xc000818640, 0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0x1, ...)
	github.com/crowdsecurity/crowdsec/pkg/parser/node.go:323 +0x151c
github.com/crowdsecurity/crowdsec/pkg/parser.Parse(0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0x0, 0x0, 0x0, ...)
	github.com/crowdsecurity/crowdsec/pkg/parser/runtime.go:288 +0xabc
main.runParse(0xc000d1b5c0, 0xc000d1b620, 0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0xc0006aca80, ...)
	github.com/crowdsecurity/crowdsec/cmd/crowdsec/parse.go:28 +0x2e8
main.runCrowdsec.func1.1(0x0, 0x0)
	github.com/crowdsecurity/crowdsec/cmd/crowdsec/crowdsec.go:55 +0x125
gopkg.in/tomb%2ev2.(*Tomb).run(0x19277e0, 0xc0003fd8e0)
	gopkg.in/tomb.v2/tomb.go:163 +0x38
created by gopkg.in/tomb%2ev2.(*Tomb).Go
	gopkg.in/tomb.v2/tomb.go:159 +0xbc
```

As mentioned in #716, manually downloading the files fixes the issue.

**Comment by sabban** [comment]
I managed to reproduce this.

It seems that `cscli hub update && cscli hub upgrade --force` do the trick. But two things worth mentioning:
* official debian documentation (in `/usr/share/doc/crowdsec`) only mention the `cscli hub update` which is fair, because it explains how enable and disable online hub back and forth
* the stack trace that happen there really shouldn't happen, and we'll now ensure this get fixed upstream.

**Comment by sabban** [comment]
I opened a new issue https://github.com/crowdsecurity/crowdsec/issues/932 for dealing with the geoip initialization failure.

The cscli behaviour should be fixed in the 1.2.0 verison (and current master).

**Comment by ThinkChaos** [comment]
Is there any solution to avoid getting in the broken state when doing `hub upgrade` on Debian?
Having to run `--force` isn't really a solution IMO.
